constructor_lab.js

```diff
@@ -0,0 +1,304 @@
+// ## Question 1
+
+// a. Write a constructor function called `Person` that has 3 properties: 
+// a first name, a last name and a middle name. Create 2 instances of a `Person`. Print one of their first names.
+
+
+function Person (firstName, lastName, middleName) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.middleName = middleName;
+}
+let syn = new Person("syn", "perez", "forrest lee");
+let maria = new Person("maria", "martinez", "liza");
+
+
+// b. Write a prototype function in `Person` called `fullName` that will return a 
+// formatted string of an instance's full name. 
+// Call this method on both the instances you created in part a.
+
+Person.prototype.fullName = function() {console.log(this.firstName + " " + this.middleName + " " + this.lastName)}
```

Comment:
This asks for you to return the string not log it. Always try to pay attention to the keywords on what you are asked to do.